libs/blocks/marquee/marquee.js

```diff
@@ -54,6 +60,22 @@ const decorateBlockBg = (block, node) => {
   }
 };
 
+function handleFocalpoint(pic, child) {
+  if (!(child)) return;
+  let text = '';
+  if (child.childElementCount == 2) {
+    text = child.querySelectorAll('p')[1]?.textContent;
+  } else if (child.textContent) {
+    text = child.textContent;
+  }
+  const image = pic.querySelector('img');
+  const directions = text.slice(text.indexOf(':') + 1).split(',');
+  const [x,y = ''] = directions
+  if (image) {
+    image.style.objectPosition = `${x.trim().toLowerCase()} ${y.trim().toLowerCase()}`;
+  }
+}
+
```

Comment:
We now have this method in two areas, here and `section-metadata.js` Is it necessary in both places? One would be preferable. 

I see you changed the authoring experience to reflect this change. Having this authored two ways could be confusing. If the requirement is only for the Marquee, then maybe remove this function from `section-metadata.js`. That being said, I believe we also want to keep `marquee.js` as small as possible since it is typically the first block to load above the fold and its images are typically set to loading eagerly prior to `loadArea`.

Reply:
The authoring experience remains the same for both marquee and section-metadata.  The background images can be added in marquee and section-metadata; both are independent. We have to author the focal point under the background images in the same way as it was done previously. I have added the function in both blocks to reduce the load on marquee and not make it dependent on section-metadata.

Reply:
For eg: https://focalpoint--milo--suhjainadobe.hlx.page/drafts/suhjain/background-image-test?martech=off
Here the authoring remains same for both section-metadata and marquee.